queen-of-versions/publish.gradle

```diff
@@ -0,0 +1,76 @@
+apply plugin: "maven-publish"
+apply plugin: "signing"
+
+task sourcesJar(type: Jar) {
+    archiveClassifier.set('sources')
+    from android.sourceSets.main.kotlin.srcDirs
+}
+
+task javadocsJar(type: Jar, dependsOn: "dokkaJavadoc") {
+    archiveClassifier.set("javadoc")
+    from dokkaJavadoc.outputDirectory
+}
```

Comment:
@NikoPelicaric did you try this? did you actually implement Dokka on PoV?

Reply:
Does dokka even work on plain java project?